x-pack/plugin/core/template-resources/src/main/resources/elastic-connectors.json

```diff
@@ -24,6 +22,9 @@
         "custom_scheduling": {
           "type": "object"
         },
+        "deleted": {
+          "type": "boolean"
+        },
```

Comment:
Does this need a rebase? I would have expected `deleted` to have been introduced in the soft-deletes PR?

Reply:
It is introduced in soft deletes. This PR moved it to its correct spot alphabetically in this list. In the soft-deletes PR it was appended to the end of the file (see code below).